adafruit_rgb_display/ssd1331.py

```diff
@@ -37,23 +42,31 @@ class SSD1331(DisplaySPI):
     """
     A simple driver for the SSD1331-based displays.
 
-    >>> import busio
-    >>> import digitalio
-    >>> import board
-    >>> from adafruit_rgb_display import color565
-    >>> import adafruit_rgb_display.ssd1331 as ssd1331
-    >>> spi = busio.SPI(clock=board.SCK, MOSI=board.MOSI, MISO=board.MISO)
-    >>> display = ssd1331.SSD1331(spi, cs=digitalio.DigitalInOut(board.GPIO0), dc=digitalio.DigitalInOut(board.GPIO15), rst=digitalio.DigitalInOut(board.GPIO16))
-    >>> display.fill(0x7521)
-    >>> display.pixel(32, 32, 0)
->>>
-from machine import Pin, HSPI
-import ssd1331
-#spi = SPI(mosi=Pin(13), sck=Pin(14), polarity=1, phase=1)
-spi = HSPI(polarity=1, phase=1)
-display = ssd1331.SSD1331(spi, dc=Pin(2), cs=Pin(15), rst=Pin(16))
-display.fill(0x7521)
-display.pixel(32, 32, 0)
+    .. code-block:: python
+
+      import busio
+      import digitalio
+      import board
+      from adafruit_rgb_display import color565
+      import adafruit_rgb_display.ssd1331 as ssd1331
+      spi = busio.SPI(clock=board.SCK, MOSI=board.MOSI, MISO=board.MISO)
+      display = ssd1331.SSD1331(spi, cs=digitalio.DigitalInOut(board.GPIO0),
+                                  dc=digitalio.DigitalInOut(board.GPIO15),
+                                  rst=digitalio.DigitalInOut(board.GPIO16))
+
+      display.fill(0x7521)
+      display.pixel(32, 32, 0)
+
+    .. code-block:: python
+
+      from machine import Pin, HSPI
```

Comment:
This is for micropython and should be removed. It should exist in the micropython driver.